group_vars/dev_slurm.yml

```diff
@@ -3,18 +3,31 @@ auth_key_user: ubuntu
 
 head_nodes: "{{ groups['dev_slurm_head'] }}"
 worker_nodes: "{{ groups['dev_workers'] }}"
+galaxy_nodes: "{{ groups['dev_galaxy_server'] }}"
+
+add_galaxy_user: true #Adds the galaxy user to all machines that have slurm on them 
 
 # SLURM
 slurm_nodes:
+    - name: dev-w1
+      NodeAddr: 115.146.85.210
+      CPUs: 4
+      RealMemory: 7780
+      State: UNKNOWN
     - name: dev-queue
       NodeAddr: 115.146.85.163
```

Comment:
This could also be written as `NodeAddr: "{{ hostvars['dev-queue']['ansible_ssh_host'] }}"`

Reply:
https://docs.ansible.com/ansible/latest/reference_appendices/special_variables.html